src/api/store/supported_language.py

```diff
@@ -2,79 +2,78 @@
 from _main_.utils.context import Context
 from _main_.utils.massenergize_errors import MassEnergizeAPIError, InvalidResourceError, NotAuthorizedError, \
     CustomMassenergizeError
+from _main_.utils.activity_logger import log
 from typing import Tuple
-from sentry_sdk import capture_message
+from django.db import IntegrityError
 
+log = log.exception
 
 class SupportedLanguageStore:
-    def __init__(self):
+    def __init__ (self):
         self.name = "Supported Language Store/DB"
 
-    def get_supported_language_info(self, context, args) -> Tuple[ dict, any ]:
+    def get_supported_language_info (self, context, args) -> Tuple[ dict or None, any ]:
         try:
             code = args.get('code', None)
-            id = args.get('id', None)
-            language = None #
-
-            if id:
-                language = SupportedLanguage.objects.filter(id=id).first()
-            elif code:
-                language = SupportedLanguage.objects.filter(code=code).first()
-            else:
+
+            if not code:
                 return None, CustomMassenergizeError("Please provide a valid id or code")
 
-            if not language:
-                return None, CustomMassenergizeError("Language not found")
+            language = SupportedLanguage.objects.filter(code = code).first()
 
             return language, None
         except Exception as e:
-            capture_message(str(e), level="error")
+            log(e)
```

Comment:
you want to do 
```suggestion
            log.exception(e)
```
do same for the others